pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "tulip"
+description = "Temporal Logic Planning (TuLiP) Toolbox"
+authors = [{name = "Caltech Control and Dynamical Systems", email = "tulip@tulip-control.org"}]
```

Comment:
We may want to change this authorship to be something more generic now. For comparison, [NumPy has "NumPy Developers"](https://github.com/numpy/numpy/blob/dc33ac4e1e59e49841f03b02098e73ba422ba56d/pyproject.toml#L17), and the [Python Control Systems Library has "Python Control Developers"](https://github.com/python-control/python-control/blob/7a70be1b7782abe0df0c69fc5d960ceb328ffbc2/pyproject.toml#L12).

Reply:
This can be fixed in the next pull request about documentation and installation.